### PR TITLE
[ Taking different formats as input data {#4}] Load data from different file formats

### PR DESCRIPTION
**Pull request by Tihsrah**
new file formats which are added are:
.txt
.json
.xlsx
.sqlite

changes are made on ClassificationDataset in datasets.py

One input is added to get table name from the sqlite in the
.sqlite file format

### CODE REVIEW

File: tab_automl/automl/datasets.py [hunk 1]
```diff
@@ -53,7 +66,19 @@ class RegressionDataset:
 
     def __init__(self, path):
         # reading the dataset from source
-        self.data = pd.read_csv(path)
+        if path[-4:] == ".txt":
```
Comment (sagnik1511) [inline tab_automl/automl/datasets.py]: Can you change this `[-some_number]` with `path.endswith` module for all the cases?
Will be a better way to understand.

File: tab_automl/automl/datasets.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 """
 This file holds all codes for custom and defined datasets.
 """
+import sqlite3
+
```
Comment (sagnik1511) [inline tab_automl/automl/datasets.py:5]: Remove this extra line with space.

File: tab_automl/automl/datasets.py (hunk 1 above)
Comment (Tihsrah) [reply tab_automl/automl/datasets.py]: Yeah sure.

File: tab_automl/automl/datasets.py [hunk 3]
```diff
@@ -14,7 +16,18 @@ class ClassificationDataset:
...
+            self.data = pd.read_table(path, delimiter='\s')
+        elif path.endswith(".json"):
+            self.data = pd.read_json(path)
+        elif path.endswith(".xlsx"):
+            self.data = pd.read_excel(path)
+        elif path.endswith(".sqlite"):
+            table_name = input("table name :")
+            db = sqlite3.connect(path)
+            self.data = pd.read_sql_query(f'Select * from {table_name}', db)
+        else:
```
Comment (sagnik1511) [inline tab_automl/automl/datasets.py]: Use elif for `csv` too.

Comment (Tihsrah) [reply tab_automl/automl/datasets.py]: done

Comment (sagnik1511) [reply tab_automl/automl/datasets.py]: Hey @Tihsrah , it is okay. Can you do this for the rest data classes? Otherwise it's perfect.

Comment (Tihsrah) [reply tab_automl/automl/datasets.py]: yes sir done (sorry i forgot to update that last time)

**Review by sagnik1511 — APPROVED** [review]

**Comment by sagnik1511** [comment]
Thanks, @Tihsrah, happy contributing!!! 🔥

**Comment by sagnik1511** [comment]
Issue #4 tagged.

**Pull request merged**